packages/phones/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@superdispatch/ui": "^0.21.14",
-    "awesome-phonenumber": "^2.70.0",
+    "awesome-phonenumber": "3.4.0",
```

Comment:
@umrzoq-toshkentov What are API changes in this version?

https://github.com/grantila/awesome-phonenumber/releases/tag/v3.0.0

Reply:
actually I need here updated libphonenumber 
https://github.com/grantila/awesome-phonenumber/compare/v3.3.0...v3.4.0

Reply:
okay I will review it